ChildToParent/childtoparent/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
```

Comment:
Please make sure to add only those folders and files that need to be reviewed. Remaining files like configs and assets should be added in .gitignore file